### DISCUSSION TITLE
Table属性Sortable等于true引发Width失效，设置Template后TextEllipsis与ShowTips失效

### DISCUSSION THREAD

**Discussion by HongFei-Ye** [dotnetcore/BootstrapBlazor#4070] [Q&A] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

![11](https://github.com/user-attachments/assets/5fc70cd1-558e-4f40-892c-3170823f2d0b)

问题：Table属性Sortable等于true引发Width失效，设置Template后TextEllipsis与ShowTips失效。

详细描述请查看复现工程(BlazorApp1.zip)

[BlazorApp1.zip](https://github.com/user-attachments/files/16620826/BlazorApp1.zip)

### Expected Behavior

复现工程内已有描述

### Steps To Reproduce

复现工程内已有描述

### .NET Version

6.0

### Anything else?

[BlazorApp1.zip](https://github.com/user-attachments/files/16620834/BlazorApp1.zip)

**Comment by bb-auto** [comment]
@HongFei-Ye Thank you for reporting. We will give triage later.

**Reply by ArgoZhang** [reply, marked as answer]
@HongFei-Ye 设置 `Template` 后很多功能基本都要自己写，因为组件无法知道模板的 DOM 结构很多样式或者功能无法做了，所以更改如下即可

```razor
<TableColumn @bind-Field="@context.AllThreadIds" Text="所有线程ID">
    <Template Context="value">
        @{
            var v = JsonSerializer.Serialize(value.Value);
        }
        <Tooltip Title="@v">
            <div style="overflow: hidden; text-overflow: ellipsis">@v</div>
        </Tooltip>
    </Template>
</TableColumn>
```
